### PR TITLE
Added support for RFC 6961 - Multiple Certificate Status Request Extension

### PR DESCRIPTION
**Pull request by filipdimitrovcom**
Added support for RFC 6961 - Multiple Certificate Status Request Extension
Fixes openssl/openssl#4449

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by t8m** [comment]
This reverts some changes on the master branch that should not be reverted such as the libssl.num changes.

**Comment by filipdimitrovcom** [comment]
Yep, sorry, my mistake, fixed now.
Probably a stupid question, but since I can't run all of your checks locally, is there a way to trigger code checks manually here in the web ("checks" tab from above)?

**Comment by t8m** [comment]
@filipdimitrovcom once some PR you've authored is merged the checks in the on pull request CI will run automatically.

File: ssl/ssl_lib.c [hunk 1]
```diff
@@ -1468,12 +1469,13 @@ void ossl_ssl_connection_free(SSL *ssl)
...
 #ifndef OPENSSL_NO_OCSP
     sk_OCSP_RESPID_pop_free(s->ext.ocsp.ids, OCSP_RESPID_free);
+    sk_OCSP_RESPONSE_pop_free(s->ext.ocsp.resp_ex, OCSP_RESPONSE_free);
 #endif
 #ifndef OPENSSL_NO_CT
     SCT_LIST_free(s->scts);
     OPENSSL_free(s->ext.scts);
 #endif
-    OPENSSL_free(s->ext.ocsp.resp);
+	OPENSSL_free(s->ext.ocsp.resp);
```
Comment (t8m) [inline ssl/ssl_lib.c:1478]: This looks like a spurious indentation change.

File: include/openssl/ocsp.h.in [hunk 2]
```diff
@@ -60,6 +60,12 @@ use OpenSSL::stackhash qw(generate_stack_macros);
 
 # ifndef OPENSSL_NO_OCSP
 
+# ifndef STACK_OF_OCSP_RESPONSE
+/* Protect against recursion */
```
Comment (t8m) [inline include/openssl/ocsp.h.in:64]: Why would this be needed? The header file is protected against recursion as a whole.

**Review by t8m — CHANGES_REQUESTED** [review]

File: include/openssl/ocsp.h.in (hunk 2 above)
Comment (filipdimitrovcom) [reply include/openssl/ocsp.h.in]: true, this code exist also for #20945, it is fixed here now

**Pull request closed without merging**